src/res/translation/translation_pt_BR.ts

```diff
@@ -4398,6 +4398,7 @@ Silenciamos seu canal e ativamos &apos;Silenciar-me&apos;. Resolva o problema de
     <message>
         <location filename="../../../mac/sound.cpp" line="463"/>
         <source>The stream format on the current input device isn&apos;t compatible with this software. Please select another device.</source>
+        <translatorcomment>English is missing %1</translatorcomment>
```

Comment:
Umm, thanks for this, we should investigate what that means in practice.

Reply:
Looks like the English text has been updated to avoid referring to `%1` or `Jamulus`. Therefore, I think the best solution is to drop the `.arg()` call. I've opened #2359 to do that.
That would mean that it would be best if you could replace `%1` by a translation of `this software` as well.

Reply:
> That would mean that it would be best if you could replace `%1` by a translation of `this software` as well.

I'm on it

Reply:
done